tests/dialects/test_builtin.py

```diff
@@ -36,9 +36,9 @@ def test_DenseIntOrFPElementsAttr_fp_type_conversion():
     value2 = check1.data.data[1].value.data
 
     # Ensure type conversion happened properly during attribute construction.
-    assert type(value1) == float
```

Comment:
I guess there cannot be subtypes of floats, right? So this is equivalent, i guess.